site2/website/versioned_docs/version-2.3.0/io-cdc-canal.md

```diff
@@ -2,6 +2,7 @@
 id: io-cdc-canal
```

Comment:
this should be `version-2.3.0-io-cdc-canal`

Reply:
> this should be `version-2.3.0-io-cdc-canal`

ok